src/content/docs/apm/agents/python-agent/supported-features/python-message-queues.mdx

```diff
@@ -17,49 +18,21 @@ import rabbitmqTransactionTraceSummary0 from 'images/rabbitmq-transaction-trace-
 
 import rabbitmqTransactionTrace from 'images/rabbitmq-transaction-trace.png'
 
-The Python agent supports the [Pika RabbitMQ client library](https://pypi.python.org/pypi/pika), giving you visibility into the performance of your message processing, for both incoming and outgoing messages.
+The Python agent supports the [Pika RabbitMQ client library](https://pypi.python.org/pypi/pika), [Kafka Python client library](https://pypi.python.org/pypi/kafka-python), and the [Confluent Kafka client library](https://pypi.org/project/confluent-kafka/) giving you visibility into the performance of your message processing, for both incoming and outgoing messages.
 
-The New Relic UI will show transactions initiated via RabbitMQ message receipt (subscribe/consume messages) as **Message** background tasks. Message creation via RabbitMQ also appears in transaction traces.
+The New Relic UI will show transactions initiated via message receipt (subscribe/consume messages) as **Message** background tasks. Message creation also appears in transaction traces.
 
 ## Requirements
 
-Requires [Python agent version 2.88.0.72 or higher](/docs/release-notes/agent-release-notes/python-release-notes).
-
-<Callout variant="caution">
-  Message tasks are not started for consumers using [TornadoConnection](https://pika.readthedocs.io/en/latest/modules/adapters/tornado.html#pika.adapters.tornado_connection.TornadoConnection).
-</Callout>
-
-## Performance improvements with background tasks [#background]
-
-One way to increase responsiveness of web applications is to delegate work to background processes. Message queues are commonly used for this inter-process communication.
-
-In the context of message queuing systems, applications typically interact with message brokers to send and receive messages. The RabbitMQ Pika client library allows Python applications to interface with message brokers that implement the Advanced Message Queueing Protocol (AMQP) 0.9 or higher.
-
-The Python agent shows messages sent and received using the RabbitMQ client library. With this visibility, you can see details including:
-
-* Number of messages produced by your app
-* Time your app spends publishing messages
-* Time your app spends processing "consumed" messages
-
-[APM](#view-queue) conveniently groups and reports operations that interact with queues. By analyzing this information, you can more easily identify bottlenecks and areas for performance improvement in your message passing architecture.
-
-## Queue operations [#queue-ops]
-
-Supported entry points for queue operations appear as `Put` (publish a message) or `Take` (receive a message) in APM's user interface.
-
 <table>
   <thead>
     <tr>
       <th width={150}>
-        **Queue operations**
-      </th>
-
-      <th>
-        **Publish a message (`Put` in UI)**
+        **Message Queue**
       </th>
 
       <th>
-        **Receive a message (`Take` in UI)**
+        **Python Agent Version Required**
```

Comment:
Good call on making this table.

Reply:
It might be helpful to extend this table to show what monitoring features we support. For example, serialization, deserialization, consumer, producer, heartbeat, etc. I'm specifically thinking of the fact that we don't support deserialization on the consumer for kafka-python here and how we might call that out. Also how we might call out/advise the customer to setup alerts for heartbeat.

Reply:
I'm thinking it might also be helpful to provide some clarification around how we expect the customer to use the consumer for each library version because we do not support other usage models. Certainly I think this would come in useful for support if nothing else. I'm not sure if we generally go into this sort of detail like this in the docs but just a thought. Something like:

In kafka-python each iteration of the consumer is reported as an individual transaction:
```
for message in consumer:
   print(message)
```

In confluent-kafka, each call to consumer.poll is reported as an individual transaction:
```
while True:
    record = consumer.poll(0.5)
    if not record:
        break
```

We could also simplify/summarize the above examples to the following:
> When consuming messages via kafka, transactions are reported for each message as it is consumed by the consumer. Transactions will not be reported for bulk consumption operations where multiple messages are consumed at once.

Reply:
In terms of the specific examples, I think we're already expecting that the customer has written code that works for them (as in, the examples we should provide are things that integrate NR APIs and such).  That being said, good call on the summary and I'll add that as a callout.

Reply:
Also, adding a separate table for features supported.